rotkehlchen/tests/api/test_misc.py

```diff
@@ -38,6 +38,27 @@ def patched_get_latest_release(_klass):
             ),
         )
 
+    result = assert_proper_response_with_result(response)
+    assert result == {
+        'version': {
+            'our_version': expected_version,
+            'latest_version': None,
+            'download_url': None,
+        },
+        'data_directory': str(rotki.data_dir),
+    }
+
+    with version_patch, release_patch:
+        response = requests.get(
+            url=api_url_for(
+                rotkehlchen_api_server,
+                'inforesource',
+            ),
+            params={
```

Comment:
interesting you went with query params instead of json, but nice to see as this is what the frontend also does